hash/nes.xml

```diff
@@ -78749,7 +78749,7 @@ We include them here as reference if they should coincide with later revisions o
 	</software>
 
 	<software name="zeldagc" cloneof="zelda">
-		<description>The Legend of Zelda (ripped from GC Zelda)</description>
+		<description>The Legend of Zelda (ripped from GameCube "Zelda")</description>
```

Comment:
Would it be a good idea to change "Gamecube "Zelda"" to "The Legend of Zelda: Collector's Edition"? That's the correct title for the disc these were ripped from.

Reply:
Yes I agree. I didn't know what was the full name.
Correction made.